python/ray/train/BUILD

```diff
@@ -749,8 +749,14 @@ py_test(
     name = "test_state_export",
     size = "small",
     srcs = ["tests/test_state_export.py"],
-    tags = ["team:ml", "exclusive"],
-    deps = [":train_lib", ":conftest"]
+    tags = [
```

Comment:
Let's not change this in this file

Reply:
Ah, it is complained by the linter in CI.